src/api/generate-xlsx/styled-columns/index.ts

```diff
@@ -51,7 +55,13 @@ const styledColumns = (application: Application, worksheet: Worksheet) => {
         wrapText: true,
       };
 
-      const isTitleRow = TITLE_INDEXES.includes(rowNumber);
+      const isInformationSheet = sheetName === APPLICATION_INFORMATION;
+      const isInformationTitleOne = isInformationSheet && rowNumber === 8;
+      const isInformationTitleTwo = isInformationSheet && rowNumber === 13;
+
+      const isInformationTitle = isInformationTitleOne || isInformationTitleTwo;
+
+      const isTitleRow = rowNumber === 1 || isInformationTitle;
```

Comment:
Created a ticket to improve/extract the logic and add unit test coverage. EMS-3539